core/src/main/java/org/infinispan/commands/remote/ClusteredGetCommand.java

```diff
@@ -78,24 +79,23 @@ public void initialize(InvocationContextFactory icf, CommandsFactory commandsFac
    public CompletableFuture<Object> invokeAsync() throws Throwable {
       // make sure the get command doesn't perform a remote call
       // as our caller is already calling the ClusteredGetCommand on all the relevant nodes
-      long flagBitSet = EnumUtil.bitSetOf(Flag.SKIP_REMOTE_LOOKUP, Flag.CACHE_MODE_LOCAL);
+      // CACHE_MODE_LOCAL is not used as it can be used when we want to ignore the ownership with respect to reads
```

Comment:
I'm not sure how the topology id could match and yet the node would not be the owner... could it happen if you receive a `ClusteredGetCommand` with topology `x` while you're still at topology `x-1`?